### DISCUSSION TITLE
MAJOR YouTube Update! Use of special characters in usernames has been limited!

### DISCUSSION THREAD

**Discussion by ThioJoe** [ThioJoe/YT-Spammer-Purge#871] [Announcements]
Official Announcement Post: https://support.google.com/youtube/thread/169359445

![image](https://user-images.githubusercontent.com/12518330/176730756-3d7f0a33-2f84-427c-ae3a-1a15ab37c1e9.png)

**Comment by Firecul** [comment]
That will be a big blow to some of the scammers, finally.

**Reply by Kurdakov** [reply]
Improved comment moderation: on Marques Brownlee channel after he set an option it worked for some time, but scam comments returned (so youtube was not about to implement good algorithm (let's say using  new Google advertised language model, which for some looked like 'real intellect' - it least it could have some chance), rather devised another simple heuristic, which was quickly broken) 
Special symbols in channel names: ok, those whatsapp numbers will just move to comments itself or Telegram accounts will be used more frequently

so youtube attempts to do something - but it looks they are just not serious to fight a problem

**Reply by Firecul** [reply]
It was the symbols I meant, surely now they can catch them... right? (/s)

**Reply by Kurdakov** [reply]
yes, currently  I see latest comments on MKBHD channel - they transferred to  refer to the telegram and also call it nicegram - less harmful, but still they are on youtube

**Reply by Firecul** [reply]
Unfortunately, there will still be many ways around any filter YT puts in place :(

**Comment by Kurdakov** [comment]
btw a result: now I remove about 200 scam comments (all telegram one) from 3-4 account on moderated channel.
actions are efficient - when scam disappear, if not - it's just trying to show an activity
